File: springboot-backend/src/main/java/backend/dto/response/AuthResponse.java
package backend.dto.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AuthResponse {
    private Long id;
    private String name;
    private String email;
    private String phone;
    private String address;
    private String avatar;
    private String role;
    private String token;
}

File: Fontend/app/src/main/java/com/example/data/dto/response/AuthResponse.kt
package com.example.data.dto.response

import com.google.gson.annotations.SerializedName

data class AuthResponse(
    @SerializedName("id")
    val id: Long,

    @SerializedName("name")
    val name: String,

    @SerializedName("email")
    val email: String,

    @SerializedName("phone")
    val phone: String?,

    @SerializedName("address")
    val address: String?,

    @SerializedName("avatar")
    val avatar: String?,

    @SerializedName("role")
    val role: String,

    @SerializedName("token")
    val token: String
)